extensions/positron-python/src/client/common/installer/moduleInstaller.ts

```diff
@@ -303,6 +303,8 @@ export function translateProductToModule(product: Product): string {
         // --- Start Positron ---
         case Product.fastapiCli:
             return 'fastapi_cli';
+        case Product.sqlite3:
+            return '_sqlite3';
```

Comment:
Is this correct?

```suggestion
            return 'sqlite3';
```

Reply:
This is what generates the original error:
https://github.com/python/cpython/blob/main/Lib/sqlite3/dbapi2.py#L27

> from _sqlite3 import *